README.md

```diff
@@ -79,6 +79,8 @@ CloudGraph AWS Provider will ask you what regions you would like to crawl and wi
 | apiGatewayResource              | apiGatewayRestApi                                                                                                                                                                                                                                                                                                                                                                                                             |
 | apiGatewayUsagePlan             |                                                                                                                                                                                                                                                                                                                                                                                                                               |
 | apiGatewayVpcLink               |                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| apiGatewayV2Api                 |                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
are you removing the service `apiGatewayHttpApi`? If so, please remove it from the readme and services list as well.

Reply:
we're keeping the same service names (I've removed the new names)
all of the new services are nested services